README.md

```diff
@@ -16,3 +16,93 @@ https://github.com/dwyl/intellectual-property and raise an
 on GitHub if you require clarification!
 
 See: https://github.com/dwyl/contributing/issues/1 + https://github.com/dwyl/summer-2015/issues/27
+
+## Recommended Guidelines
+
+### Labels
+
+GitHub can be a _great_ project management tool if used _effectively_. One of the ways you can manage your project backlog is through **labels**. Labels are a great way to **organise** and **track** the progress of a given project at _any_ point in time.
+
+At dwyl we have created our _own_ standard list of labels that we use for each of our projects. This gives our developers the ability to move between projects **quickly** and **productively**, saving time and energy in the process.
+
+Here is our **[complete list](https://github.com/dwyl/contributing/labels)** of labels and how they should be used:
+
+![labels](https://cloud.githubusercontent.com/assets/12450298/18248682/afcd6974-7371-11e6-84bf-0cb9f4677d92.png)
+
+- `bug` #EE0701 - report a bug that needs fixed
```

Comment:
Might be good to split the default GH ones (`bug`, `duplicate`, `enhancement`, `help-wanted`, `invalid`, `wontfix` and `question`)  from the dwyl ones?

Just an idea, not even sure it's a good one!